pkg/integrations/node_exporter/config.go

```diff
@@ -80,61 +80,61 @@ func init() {
 
 // Config controls the node_exporter integration.
 type Config struct {
```

Comment:
This should really be broken up into multiple blocks (one subblock per configurable collector)  since it's such a massive list of attributes, but that can be a follow up change.